.github/workflows/update-snapshot.yml

```diff
@@ -0,0 +1,28 @@
+name: Update emacs snapshots with flake
+
+on:
+  schedule:
+    - cron: "15 10 * * 2"
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: cachix/install-nix-action@v21
+      with:
+        github_access_token: ${{ secrets.GITHUB_TOKEN }}
+        nix_path: nixpkgs=channel:nixos-unstable
```

Comment:
We don't need this line because we only run `nix flake update` which doesn't depend a nixpkgs channel.